README.md

```diff
@@ -1 +1,21 @@
-# dcos-cni-plugins
+# Purpose
+This repo hosts the CNI plugins specific to DC/OS. A brief description
+of each of the plugins is given below:
+
+* [plugins/l4lb](plugins/l4lb/README.md): A CNI plugin which allows containers in isolated virtual networks to use services provided by [Minuteman](https://github.com/dcos/minuteman) and [Spartan](https://github.com/dcos/spartan).
```

Comment:
.. and i'd call it `cmd` since it contains executable binary "commands".
xref https://peter.bourgon.org/go-best-practices-2016/